src/status_im/signing/core.cljs

```diff
@@ -107,20 +108,22 @@
   (log/debug "[signing] prepare-unconfirmed-transaction")
   (let [token (tokens/symbol->token (:wallet/all-tokens db) symbol)
         from  (eip55/address->checksum from)]
-    {:db (assoc-in db [:wallet :accounts from :transactions hash]
-                   {:timestamp (str now)
-                    :to        to
-                    :from      from
-                    :type      :pending
-                    :hash      hash
-                    :data      data
-                    :token     token
-                    :symbol    symbol
-                    :value     (if token
-                                 (money/unit->token amount (:decimals token))
-                                 (money/to-fixed (money/bignumber value)))
-                    :gas-price (money/to-fixed (money/bignumber gasPrice))
-                    :gas-limit (money/to-fixed (money/bignumber gas))})}))
+    {:db (-> db
+             (update-in [:wallet :accounts from :transactions] dissoc (:hash tx))
```

Comment:
It's not clear what's the difference between `tx` and `hash`, we are `dissocing` first `(:hash tx)` and then `assoc` `hash` back in, I suppose that `hash` is the new hash while `(:hash tx)` is the old one? 
If so maybe we could make it clearer from the parameter names.

Reply:
yeah good, point, also was thinking about that

Reply:
@cammellos done